pygls/types.py

```diff
@@ -1270,8 +1282,8 @@ def __init__(self, id: str, method: str):
 
 
 class UnregistrationParams:
-    def __init__(self, unregistrations: List[Unregistration]):
-        self.unregistration = unregistrations
+    def __init__(self, unregisterations: List[Unregistration]):
```

Comment:
Renamed unregistrations to unregisterations. Was this accidental?

Reply:
It was really hard to find why it wasn't working and the reason was that [specification](https://microsoft.github.io/language-server-protocol/specification#client_unregisterCapability) says "unregisterations" (!?) :)

Reply:
That must've been frustrating.